community/examples/intel/README.md

```diff
@@ -1,5 +1,8 @@
 # Intel Solutions for the HPC Toolkit
 
+> **_NOTE:_** The [hpc-slurm-daos.yaml](hpc-slurm-daos.yaml) will not be compatible
+> for newer version of slurm-gcp v6.
```

Comment:
Somewhere, probably in PR description, we should say that alt solution would be to upgrade daos to use TPG 5.x.

Reply:
Added in the PR desc.